modulemd/v2/tests/test-modulemd-modulestream.c

```diff
@@ -616,6 +616,54 @@ module_stream_v2_test_depends_on_stream (ModuleStreamFixture *fixture,
   g_clear_object (&stream);
 }
 
+static void
+module_stream_test_community (ModuleStreamFixture *fixture,
+                                         gconstpointer user_data)
+{
+  g_autoptr (ModulemdModuleStream) stream = NULL;
+  guint64 version;
+  g_autofree gchar *community = NULL;
+  
+  for (version = MD_MODULESTREAM_VERSION_ONE;
+       version <= MD_MODULESTREAM_VERSION_LATEST;
+       version++)
+  {
+    stream = modulemd_module_stream_new (version, "foo", "latest");
+    g_assert_nonnull (stream);
+    
+    g_assert_null (modulemd_module_stream_get_community(stream));
+    
+    // clang-format-off
+    g_object_get (stream,
+                    "community", &community,
```

Comment:
There are numerous formatting errors in this file. Many of them would be fixed by running `ninja test`, which also calls `clang-format` automatically. However, the sections between `// clang-format-off` and `// clang-format on` do not get auto-formatted and need to be aligned properly on their own.

It looks here like you copied this from some other example and did not adjust the whitespace accordingly.

Reply:
The same is true in each place you have `// clang-format off`